src/Employee.java

```diff
@@ -1,3 +1,4 @@
+package src;
```

Comment:
# Remove the "package src ;" 
####  I am guessing that you were running into issues compiling the code. I understand the error and have ran into this issue. 
 - Simply remove this line
 - quit and restart your vs code on your mac
 -  then reopen the project and it should work.
 ## Reasoning 
 - This is a vscode error that is assuming your root to be the highest file level that any java files have been in relative to the dir that you're in. When you restart your vs code and manually reopen your project in vs code, then it should work.